modules/installing-rhv-requirements.adoc

```diff
@@ -30,6 +30,8 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 *** 16 GiB or more for each of the three control plane machines which provide the control plane.
 *** 16 GiB or more for each of the three compute machines, which run the application workloads.
 * The {rh-virtualization} storage domain must meet link:https://access.redhat.com/solutions/4770281[these etcd backend performance requirements].
+* 1 available physical machine per worker and 1 available host per control plane. These can both be on the same host.
+* To use hard anti-affinity, you need a minimum of 3 phys machines, but if you have more than three worker nodes, then as many phys machines as you have worker nodes.
```

Comment:
To use enforcing affinity groups one physical machine will be required for each virtual machine in the affinity group.